README.md

```diff
@@ -23,47 +23,6 @@ Visit [charmed-kubeflow.io][charmedkf] for more information.
 
 For any Kubernetes, follow the [installation instructions][install].
 
-## Testing
-
```

Comment:
suggestion(non-blocking): We could have a `## Testing` section that would point to our UATs. WDYT? sth very simple that would just refer to the repo.

Reply:
whatever you decide, let's do the same for those READMEs as well since we 're still using them https://github.com/canonical/bundle-kubeflow/blob/d502f283b2713f0310bd2b25825ecc2f74f4b30f/releases/latest/edge/README.md https://github.com/canonical/bundle-kubeflow/blob/d502f283b2713f0310bd2b25825ecc2f74f4b30f/releases/latest/beta/README.md